neo/Persistence/StoreView.cs

```diff
@@ -0,0 +1,94 @@
+using Neo.IO.Caching;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Persistence
+{
+    public abstract class StoreView
+    {
```

Comment:
I think that this should be done outside `Blockchain` here is a good place for it

https://github.com/neo-project/neo/blob/8ccc5e93d82d534dafa2ff8dd751b9817f80d09b/neo/Ledger/Blockchain.cs#L62

Reply:
This is not the problem that this PR has to solve.